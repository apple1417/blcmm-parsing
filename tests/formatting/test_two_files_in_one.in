<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="file 1" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="First File">
			<comment>A B C</comment>
		</category>
	</body>
</BLCMM>

First Command Block

<BLCMM v="1">
#<!!!You opened a file saved with BLCMM in FilterTool. Please update to BLCMM to properly open this file!!!>
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="file 2" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="Second File">
			<code profiles="">set d e f</comment>
		</category>
	</body>
</BLCMM>

Second Command Block
