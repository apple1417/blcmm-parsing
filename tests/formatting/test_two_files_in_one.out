<BLCMM v="1">
	<head>
		<type name="BL2" offline="false"/>
		<profiles>
			<profile name="file 1" current="true"/>
		</profiles>
	</head>
	<body>
		<category name="First File">
			<comment>A B C</comment>
		</category>
	</body>
</BLCMM>
